KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/KeywordSearchIngestModule.java

```diff
@@ -807,4 +814,27 @@ private boolean indexTextFile(AbstractFile aFile) {
             return false;
         }
     }
+    
+    /**
+     * This class takes a list of ProcessTerminators checking all of them
+     * during shouldTerminateProcess.
+     */
+    private class HybridTerminator implements ProcessTerminator {
```

Comment:
Very nice use of composition :)

Reply:
It might make sense to move this to the ProcessTerminator for better reuse, but I was keeping the scope small for this story.